test/unit/mailers/provider_user_mailer_test.rb

```diff
@@ -91,9 +90,7 @@ def setup
         assert_equal '{"category": "Signup Notification"}', email.header_fields.find{ |header| header.name.eql? 'X-SMTPAPI' }.value
 
         assert_match %r{Dear #{user.informal_name}}, email_body
-        assert_match %r{Thank you for joining #{master_account.org_name}}, email_body
         assert_match %r{#{account.admin_domain}/p/activate/[a-z0-9]+}, email_body
-        assert_match %r{The #{master_account.org_name} team}, email_body
```

Comment:
we cannot just remove tests

Reply:
I thought it was checking for a variable, and since there's no variable now...

Reply:
I put them back (hopefully everywhere they are needed)